File: Some_Important_Topics/Fibonacci_Series.c
// Fibonacci Series With Recursion (Final limit digit sum)

#include <stdio.h>

int Fibo(int limit);

int main()
{
    int limit;
    printf("Enter the limit : ");
    scanf("%d",&limit);
    int result = Fibo(limit);
    printf("Final Result : %d\n",result);
    return 0;
}
int Fibo(int limit){
    if(limit<=1){
        return 1;
    }
    return Fibo(limit-1)+Fibo(limit-2);
}

File: Some_Important_Topics/Fibonacci_Series.js
// Fibonacci Series With Recursion

const limit = 5;

const Fibo=(limit)=>{
    if(limit<=1){
        return 1;
    }
    return Fibo(limit-1)+Fibo(limit-2);
}

const result = Fibo(limit);
console.log(`Final Result : ${result}`);